bq-connector/docai_bq_connector/doc_ai_processing/Processor.py

```diff
@@ -213,38 +224,53 @@ def _process_async(self) -> Union[DocumentOperation, ProcessedDocument]:
         storage_client = storage.Client()
         bucket = storage_client.get_bucket(output_bucket)
         blob_list = list(bucket.list_blobs(prefix=prefix))
-        logging.debug("Output files:")
         # should always be a single document here
         for i, blob in enumerate(blob_list):
             # If JSON file, download the contents of this blob as a bytes object.
             if blob.content_type == "application/json":
                 blob_as_bytes = blob.download_as_bytes()
 
                 document = documentai.types.Document.from_json(blob_as_bytes)
```

Comment:
```suggestion
                document = documentai.Document.from_json(blob_as_bytes, ignore_unknown_fields=True)
```

Reply:
I added this change to the samples https://cloud.google.com/document-ai/docs/send-request#async-processor because the API might update the Document response with new fields and if you don't have the latest client library version installed, then this function would throw an exception. This extra parameter provides a safety net